src/Components/Aside/index.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 
-export default class Aside extends Component {
+export default class index extends Component {
```

Comment:
```suggestion
export default class Aside extends Component {
```
컴포넌트 이름은 대문자로 시작하고, index 자체는 아무런 뜻을 갖고 있지 않기 때문에 이름도 Aside로 그대로 유지해주시는것이 맞는 것 같습니다